### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/deal-making-issues.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Please provide all the information requested here to help us troubleshoot "deal making failed" issues.
+If the information requested is missing, we will probably have to just ask you to provide it anyway,
+before we can help debug.
+
+**Basic Information**
+Including information like, Are you the client or the miner? Is this a storage deal or a retrieval deal? Is it an offline deal?
+
+**Describe the problem**
+
+A brief description of the problem you encountered while deal making.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/deal-making-issues.md:19]:
```suggestion
A brief description of the problem you encountered while trying to make a deal. 
```

File: .github/ISSUE_TEMPLATE/deal-making-issues.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+A brief description of the problem you encountered while deal making. 
+
+**Version**
+
+The output of `lotus --version`.
+
+**Setup**
+
+You miner(if applicable) and daemon setup.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/deal-making-issues.md:27]: Should probably elaborate a bit more on what "setup" means in this context

File: .github/ISSUE_TEMPLATE/deal-making-issues.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+**Commands**
+
+Commands you ran.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/deal-making-issues.md:31]: I'd use the 'To Reproduce' from bug-report.md, you don't have to run commands to make a deal

File: .github/ISSUE_TEMPLATE/deal-making-issues.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+**Deal status**
+
+The output of `lotus client` or `lotus-miner storage-deals|retrieval-deals|data-transfers` commands for the deal(s) in question.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/deal-making-issues.md:35]:
```suggestion
The output of `lotus client list-deals -v` and/or `lotus-miner storage-deals|retrieval-deals|data-transfers list [-v]` commands for the deal(s) in question.
```

File: .github/ISSUE_TEMPLATE/mining-issues.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+about: Create a report for help with mining failures.
+title: "[Mining Issue]"
+labels: hint/needs-triaging, area/mining
+assignees: ''
+
+---
+
+Please provide all the information requested here to help us troubleshoot "mining/WinningPoSt failed" issues.
+If the information requested is missing, we will probably have to just ask you to provide it anyway,
+before we can help debug.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/mining-issues.md:12]:
```suggestion
If the information requested is missing, you may be asked you to provide it.
```

File: .github/ISSUE_TEMPLATE/proving-issues.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+---
+name: Proving Issues
+about: Create a report for help with proving failures.
+title: "[Proving Issue]"
+labels: area/proving, hint/needs-triaging
+assignees: ''
+
+---
+
+Please provide all the information requested here to help us troubleshoot "proving/wdpost failed" issues.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/proving-issues.md:10]:
```suggestion
Please provide all the information requested here to help us troubleshoot "proving/window PoSt failed" issues.
```

File: .github/ISSUE_TEMPLATE/deal-making-issues.md (hunk 2 above)
Comment (jennijuju) [reply .github/ISSUE_TEMPLATE/deal-making-issues.md]: updated

File: .github/ISSUE_TEMPLATE/deal-making-issues.md (hunk 3 above)
Comment (jennijuju) [reply .github/ISSUE_TEMPLATE/deal-making-issues.md]: updated

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Provide daemon/miner/worker and goroutines(if available) for troubleshooting.
+ 
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (Kubuxu) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: Would be good to mention that security-related bugs/issues should follow the security issue track.

Comment (jennijuju) [reply .github/ISSUE_TEMPLATE/bug-report.md:33]: updated

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Run '...'
+2. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Logs**
+Provide daemon/miner/worker and goroutines(if available) for troubleshooting.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/bug-report.md:27]:
```suggestion
Provide daemon/miner/worker logs, and goroutines(if available) for troubleshooting.
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: hint/needs-triaging, kind/bug
+assignees: ''
+
+---
+
+> Note: For security-related bugs/issues, please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/bug-report.md:10]: won't this be visible in the issue text?

File: .github/ISSUE_TEMPLATE/proving-issues.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+The output of `lotus --version`.
+
+**Setup**
+
+You miner and daemon setup, including what hardware do you use, your environment variable settings, how do you run your miner and worker, do you use GPU and etc.
+
+**Commands**
+
+Commands you ran.
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/proving-issues.md:29]: Not sure if this applies to PoSting issues that much

File: .github/ISSUE_TEMPLATE/proving-issues.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+Please go through the logs of your miner, and include screenshots of any error-like messages you find, highlighting the one has "window post" in it.
+
+Alternatively please upload full log files and share a link here
+
+**Lotus miner diagnostic info**
+
+Please collect the following diagnostic information, and share a link here
+
+* lotus-miner diagnostic info `lotus-miner info all > allinfo`
```
Comment (magik6k) [inline .github/ISSUE_TEMPLATE/proving-issues.md:45]: I'd do
```suggestion
* lotus-miner diagnostic info `lotus-miner info all > allinfo.txt`
```
so it's painfully obvious that the output is saved to a file

**Review by magik6k — COMMENTED** [review]
**Describe the bug**
There are some issues in this PR

**Version (run `lotus version`):**
zsh: command not found: lotus

**To Reproduce**
Read review comments

**Expected behavior**
No bugs in issue templates

**Logs**
[here](https://github.com/filecoin-project/lotus/pull/5544/files)
 
**Screenshots**
n/a

**Additional context**
Generally looks good, just a couple of nitpics

File: .github/ISSUE_TEMPLATE/proving-issues.md (hunk 11 above)
Comment (jennijuju) [reply .github/ISSUE_TEMPLATE/proving-issues.md:45]: LOL

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 9 above)
Comment (jennijuju) [reply .github/ISSUE_TEMPLATE/bug-report.md:10]: image
yes!

**Review by magik6k — APPROVED** [review]

**Pull request merged**